pom.xml

```
@@ -155,7 +155,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-2.387.x</artifactId>
-        <version>2230.v0cb_4040cde55</version>
+        <version>2483.v3b_22f030990a_</version>
```

Last release of the 2.387.x bom
```suggestion
        <version>2543.vfb_1a_5fb_9496d</version>
```